sdk/eventhub/azure-eventhub-checkpointstoretable/README.md

```diff
@@ -0,0 +1,40 @@
+[![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/azure-sdk-for-python.client?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=46?branchName=master)
```

Comment:
We've migrated the `master` branch to `main`, so references to `master` should be updated.
```suggestion
[![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/azure-sdk-for-python.client?branchName=main)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=46?branchName=main)
```